### ISSUE TITLE
more panel

### ISSUE THREAD

**Issue by wifi75** [stepsolar/hassio-package-panel-solar#2]
Hello I have 23 pannel I tried to add them to my dashboard, but I only ever see 8, why?

![image](https://user-images.githubusercontent.com/31816231/232439985-22d0d850-281a-44c4-a25e-2ced06fbe401.png)

**Comment by DeFlanko** [comment]
i ran into the same issue -- it has to do with the back end codes only have 8 because the SVG that's used is only that layout -- so now im making my OWN SVG layout based on how they're lined up on the roof and then hopefully will eb able to post some steps on how to do this.

**Comment by rogerthn2019** [comment]
Looks like we are 2 that have this challenge ;-)
[My layout](https://monitoringpublic.solaredge.com/solaredge-web/p/site/public?name=Gokottevagen1&locale=en_US#/layout)
Currently in HA
![image](https://user-images.githubusercontent.com/57598997/233831612-24afa046-99c1-48c5-9c38-8f196ceb66ce.png)
...

**Comment by stepsolar** [comment]
you can change the layout with inkscape

**Comment by rogerthn2019** [comment]
Realized that it might be hard to mimic [my layout](https://monitoringpublic.solaredge.com/solaredge-web/p/site/public?name=Gokottevagen1&locale=en_US#/layout) and that I need to calculate todays production, a utility meter helper should do the trick.
Using glance card as below I do have
![image](https://user-images.githubusercontent.com/57598997/233837171-f3aa7582-6754-4568-8fe8-9dbafcb9db1b.png)

`cards:`
`  - type: glance`
`    entities:`
`      - entity: sensor.last_measurement_1_1_4`
`        name: Last update`
`      - entity: sensor.lifetime_energy_1_1_4`
`        name: Lifetime`
`      - entity: sensor.optimizer_voltage_1_1_4`
`        name: Optimizer`
`      - entity: sensor.current_1_1_4`
`        name: Current`
`      - entity: sensor.power_1_1_4`
`        name: Power`
`      - entity: sensor.voltage_1_1_4`
`        name: Voltage`
`    title: A P01 1.1.4`
`    show_icon: false`
`    columns: 3`
`  - type: glance`
`    entities:`
`      - entity: sensor.last_measurement_1_1_28`
`        name: Last update`
`      - entity: sensor.lifetime_energy_1_1_28`
`        name: Lifetime`
`      - entity: sensor.optimizer_voltage_1_1_28`
`        name: Optimizer`
`      - entity: sensor.current_1_1_28`
`        name: Current`
`      - entity: sensor.power_1_1_28`
`        name: Power`
`      - entity: sensor.voltage_1_1_28`
`        name: Voltage`
`    title: A P02 1.1.28`
`    show_icon: false`
`    columns: 3`

**Comment by DeFlanko** [comment]
> Looks like we are 2 that have this challenge ;-) [My layout](https://monitoringpublic.solaredge.com/solaredge-web/p/site/public?name=Gokottevagen1&locale=en_US#/layout) Currently in HA ![image](https://user-images.githubusercontent.com/57598997/233831612-24afa046-99c1-48c5-9c38-8f196ceb66ce.png) ...

Interesting! your layout looks like a real layout on a roof. Where mine.. is just a [strait line](https://monitoringpublic.solaredge.com/solaredge-web/p/site/public?name=DeFlanko#/dashboard)! and the first 24 of them were OLD panels.. they never removed them from the layout...(i for some reason cannot share like you did it only shared the dashboard) 

![image](https://user-images.githubusercontent.com/5233969/233852329-3eb46bcd-e412-4a1a-bef4-d2928a1fe8ac.png)


Do you have access to adjust the layout of the panels on the solar edge site? id love to clean mine up to look more like how it is [on the roof](https://github.com/DeFlanko/hassio-package-panel-solar).

Also how are you you getting values back into the HA? im just unable to get values from Solar Edge back in to Ha...

**Comment by rogerthn2019** [comment]
The layout was done by the installer, maybe ask SolarEdge support?
I did install [SolarEdge Optimizers Data](https://github.com/ProudElm/solaredgeoptimizers) and after some time  I had all optimizer in HA.
Renamed to mimic my layout A=top row, B=middle row and C=bottom, Pnn=position number.
Using glance card and HACS layout-card I now have a dashboard with data from all my optimizers
![image](https://user-images.githubusercontent.com/57598997/233855222-cd77d730-3889-406c-9ef8-ac53d60fe3a2.png)
The daily values is not for a full day since I did configure Utility Meter helpers after lunch
